### DISCUSSION TITLE
560_sap/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#627] [Announcements]
# 560_sap/

前言 本手册是五六零网校为培养出优秀的 SAP Business One 实施顾问而编写的学习手册。 我们的宗旨是为学员提供系统、全面、实用的 SAP Business One 实施顾问培训，帮助学员掌握 SAP Business One 实施的基础知识和实践技能。 学习方式 学员可以在网校免费学习，网校会提供相关学习资源。; 网校有学员讨论群，仅供学...

https://help.sap560.com/560_sap/

**Comment by 560e** [comment]
欢迎来到五六零网校

**Reply by Mister-Hope** [reply]
Do not use my repo in production.  This is a final warning.
